<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>image</class>
 <widget class="QDialog" name="image">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1305</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>851</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>370</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QDial" name="dial">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>200</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-360</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
  </widget>
  <widget class="QLabel" name="sliderValueLBL">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>370</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="chemin_2">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>490</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="dialValueLBL">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>250</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="left_LBL">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>570</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Left Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="rigth_LBL">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>570</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Right Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="ctrl_LBL">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>610</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>CTRL Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="pic">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1301</width>
     <height>801</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>430</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:checked {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,   stop:0 rgb(120, 201, 255), stop:1 rgb(120, 0, 255))
}

QPushButton:hover {

	background-color: rgb(170, 255, 255);
}
QPushButton {
     background-color: rgb(120, 201, 255); border: 1px solid black;
	border-color: rgb(170, 170, 255);
     border-radius: 5px;
}</string>
   </property>
   <property name="text">
    <string>Convertir Image</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>430</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
padding: 5px;
border-radius: 5px;

font-weight: bold;</string>
   </property>
   <item>
    <property name="text">
     <string>.png</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.bmp</string>
    </property>
   </item>
  </widget>
  <action name="actionZoomInOut">
   <property name="text">
    <string>ZoomInOut</string>
   </property>
   <property name="toolTip">
    <string>Zoom Image</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="text">
    <string>Rotate</string>
   </property>
   <property name="toolTip">
    <string>Ruot</string>
   </property>
  </action>
  <zorder>pic</zorder>
  <zorder>graphicsView</zorder>
  <zorder>horizontalSlider</zorder>
  <zorder>dial</zorder>
  <zorder>sliderValueLBL</zorder>
  <zorder>chemin_2</zorder>
  <zorder>dialValueLBL</zorder>
  <zorder>left_LBL</zorder>
  <zorder>rigth_LBL</zorder>
  <zorder>ctrl_LBL</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>comboBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
